Dockerfiles/agent/amd64/Dockerfile

```diff
@@ -175,7 +184,15 @@ HEALTHCHECK --interval=30s --timeout=5s --retries=2 \
 # Leave following directories RW to allow use of kubernetes readonlyrootfs flag
 VOLUME ["/var/run/s6", "/var/log/datadog"]
 
-CMD ["/init"]
+# Ensure the glibc doesn't try to call the `faccessat2` syscall
+COPY --from=faccessat /tmp/libfaccessat.so /lib/x86_64-linux-gnu/libfaccessat.so
+ENV LD_PRELOAD /lib/x86_64-linux-gnu/libfaccessat.so
```

Comment:
Does using `LD_PRELOAD` cause problems with SELinux and/or seccomp?

Reply:
It has no interaction since SELinux and seccomp are kernel features whereas `LD_PRELOAD` is a pure userland mechanism.
From the `glibc` point of view, honoring `LD_PRELOAD` doesn’t require to invoke different system calls than the ones needed to load a “regular” library.

I needed this library specially for the `system-probe` container because of its seccomp profile that might miss the `faccessat2` system call if the `libseccomp` of the host is too old.
So, I tested and validated that this `LD_PRELOAD` trick works for `system-probe` with a seccomp profile loaded.